robottelo/hosts.py

```diff
@@ -1052,17 +1052,6 @@ def configure_rhai_client(
         :param register: Whether to register client to insights
```

Comment:
Could you use the same PR to describe use of `register_insights` param here?

Reply:
its a setting when registering a client to a sat. You can set up insights during this process. This is true by default

Reply:
@ColeHiggins2 yes, but I meant if it could be described in the function metadata, which it isn't